2023-01-04 21:41:05,974 (40018ab808) :  INFO (core:349) - Starting framework core - Version: 2.6.3, Build: 629d580 (Thu Dec 08 08:44:59 UTC 2022)
2023-01-04 21:41:06,139 (40018ab808) :  DEBUG (core:361) - Using the elevated policy
2023-01-04 21:41:06,187 (40018ab808) :  DEBUG (core:450) - Starting runtime component.
2023-01-04 21:41:06,233 (40018ab808) :  DEBUG (core:450) - Starting caching component.
2023-01-04 21:41:06,244 (40018ab808) :  DEBUG (core:450) - Starting data component.
2023-01-04 21:41:06,254 (40018ab808) :  DEBUG (core:450) - Starting networking component.
2023-01-04 21:41:06,626 (40018ab808) :  DEBUG (networking:261) - Loaded HTTP cookies
2023-01-04 21:41:06,711 (40018ab808) :  DEBUG (caching:108) - No info file found, trashing the cache folder
2023-01-04 21:41:06,940 (40018ab808) :  DEBUG (networking:428) - Setting the default network timeout to 20.0
2023-01-04 21:41:06,990 (40018ab808) :  DEBUG (core:450) - Starting localization component.
2023-01-04 21:41:06,999 (40018ab808) :  INFO (localization:409) - Setting the default locale to en-us
2023-01-04 21:41:07,008 (40018ab808) :  DEBUG (core:450) - Starting messaging component.
2023-01-04 21:41:07,021 (40018ab808) :  DEBUG (core:450) - Starting debugging component.
2023-01-04 21:41:07,031 (40018ab808) :  DEBUG (core:450) - Starting services component.
2023-01-04 21:41:07,039 (40018ab808) :  DEBUG (core:450) - Starting myplex component.
2023-01-04 21:41:07,042 (400db3bb38) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/system/messaging/clear_events/com.plexapp.agents.lastfm'
2023-01-04 21:41:07,058 (40018ab808) :  DEBUG (core:450) - Starting notifications component.
2023-01-04 21:41:08,080 (40018ab808) :  DEBUG (accessor:68) - Creating a new model access point for provider com.plexapp.agents.lastfm in namespace 'metadata'
2023-01-04 21:41:08,128 (40018ab808) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes'
2023-01-04 21:41:08,167 (40018ab808) :  ERROR (networking:196) - Error opening URL 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes'
2023-01-04 21:41:08,177 (40018ab808) :  CRITICAL (runtime:1299) - Exception getting hosted resource hashes (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-629d58034/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 1293, in get_resource_hashes
    json = self._core.networking.http_request("http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes", timeout=10).content
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-629d58034/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 242, in content
    return self.__str__()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-629d58034/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 220, in __str__
    self.load()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-629d58034/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 158, in load
    f = self._opener.open(req, timeout=self._timeout)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 435, in open
    response = meth(req, response)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 548, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 473, in error
    return self._call_chain(*args)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 556, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 404: Not Found

2023-01-04 21:41:08,226 (400e7eab38) :  DEBUG (services:265) - Plug-in is not daemonized - loading services from system
2023-01-04 21:41:08,227 (40018ab808) :  DEBUG (runtime:1117) - Created a thread named 'load_all_services'
2023-01-04 21:41:08,242 (40018ab808) :  DEBUG (runtime:1117) - Created a thread named 'get_server_info'
2023-01-04 21:41:08,248 (400e7eab38) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/messaging/function/X0J1bmRsZVNlcnZpY2U6QWxsU2VydmljZXM_/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMApyMAo_'
2023-01-04 21:41:08,250 (40018ab808) :  DEBUG (core:150) - Finished starting framework core
2023-01-04 21:41:08,256 (400e837b38) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400'
2023-01-04 21:41:08,259 (40018ab808) :  DEBUG (core:560) - Loading plug-in code
2023-01-04 21:41:08,428 (400e7eab38) :  DEBUG (services:362) - Loaded services
2023-01-04 21:41:08,446 (400e837b38) :  DEBUG (core:538) - Machine identifier is 9449cbc9ca788c4ad1934f7dda63226ffd1e1d27
2023-01-04 21:41:08,465 (400e837b38) :  DEBUG (core:539) - Server version is 1.30.0.6486-629d58034
2023-01-04 21:41:08,476 (400e814b38) :  DEBUG (services:438) - No shared code to load
